src/sagemaker_mxnet_container/serving.py

```diff
@@ -284,9 +289,12 @@ def main(environ, start_response):
     global app
     if app is None:
         serving_env = env.ServingEnv()
+        _update_mxnet_env_vars()
```

Comment:
why wasn't this needed before?

Reply:
it was...I forgot it in my last PR :sob:

Reply:
cool